website/blog/2021-06-13-Hydra_1.1.md

```diff
@@ -0,0 +1,129 @@
+- A top level "experiment config" can now override the Defaults List as well as config values
+- Complex frameworks can now have their own Defaults List, reducing boilerplate
+
+Other related changes includes the ability to change the order a config is composed relative to config in its Defaults List by 
+adding `_self_` to the Defaults List and the ability to use multiple configs from the same config group.
+
+Learn more:
+- [The Defaults List](/docs/next/advanced/defaults_list)
+- [Extending configs](/docs/next/patterns/extending_configs)
+- [Structured Configs Schema](/docs/next/tutorials/structured_config/schema)
+- [Configuring Experiments](/docs/next/patterns/configuring_experiments)
+- [Select multiple configs from config group](/docs/next/patterns/select_multiple_configs_from_config_group)
+
+### Object instantiation enhancements
+`hydra.utils.instantiate()` now instantiate nested objects recursively.
+
+Other enhancements includes:
+- Support for positional arguments via the `_args_` config key
+- Support to parameter conversion strategy was added via the `_convert_` config key.
+
+Learn more [here](/docs/next/advanced/instantiate_objects/overview).
+
+### Hydra callbacks
+A new experimental mechanism for user defined callbacks was added.
+Callbacks enables user code to be executed automatically at various points in the lifecycle of your application. 
+There are many potential use cases for this, for example automatic registration with your 
+favorite experiment tracking service.
+
+Learn more [here](/docs/next/experimental/callbacks).
+
+### OmegaConf 2.1 highlights
+OmegaConf 2.1 includes many enhancements, bug fixes, and performance improvements.
+
+#### Relative interpolations
+Relative interpolations enable accessing a config node relative to the node defining the interpolation:
+```yaml
+x: 10
+b:
+  y: 20
+  a: {x}    # 10, absolute interpolation
+  d: ${.y}  # 20, relative interpolation
+  e: ${..x} # 10, relative interpolation
+```
+
+#### Nested interpolations
+OmegaConf 2.1 adds a new interpolation grammar supporting more sophisticated usage of interpolations.  
+In the following example, the default value to use if the environment variable `DB_USER` does not exist is defines in the `default_user` config node:  
+```yaml
+default_user: root
+db_user: ${oc.env:DB_USER,${default_user}}
+```
+
+### More powerful custom resolvers
+OmegaConf custom resolvers can now access parent config node or the config root by defining
+keyword parameters named `_parent_` and `_root_`.
+
+In the example below, we use `_parent_` to implement a sum function that defaults to 0 if the node does not exist:
+
+```python
+def sum(a, b, *, _parent_):
+  return _parent_.get(a, 0) + _parent_.get(b, 0)
+
+OmegaConf.register_new_resolver("sum", sum, use_cache=False)
+cfg = OmegaConf.create({
+  "node": {
+    "a": 1,
+    "b": 2,
+    "a_plus_b": "${sum:a,b}",
+    "a_plus_z": "${sum:a,z}",
+  },
+})
```

Comment:
is it possible to include `_root_` in the example as well? 
Also there's no example of `_root_` in omegaconf's doc, could we add it?

Reply:
It's not important enough for an example.